src/test/java/plugins/AnalysisCollectorPluginTest.java

```diff
@@ -329,6 +330,33 @@ public void should_aggregate_warnings_in_dashboard_portlet() {
         assertThat(dashboard, not(hasWarningsFor(job, TASKS, TASKS_ALL)));
     }
 
+    @Test
+    @WithPlugins({"git@2.4.0", "workflow-aggregator@1.10"})
```

Comment:
Oops, git was used in the previous approach (there was a git clone as a step). I have to delete it. And a concrete WF version is not required neither, but I think it's a good practice to specify the version, don't you think so?